doc/authoring_command_modules/README.md

```diff
@@ -199,28 +199,36 @@ azdev style --module <module> [--pylint] [--pep8]
+  * `[]` means this change is customer-facing and the message will be put into `HISTORY.rst`. `{}` means this change is not customer-facing and the message will **NOT** be included in `HISTORY.rst`. 
+  * If the component name is `Core`, the message will be written in `src/azure-cli-core/HISTORY.rst`. Otherwise, the message will be written in `src/azure-cli/HISTORY.rst`.
+2. [**Mandatory**] If it's a breaking change, the second part should be `BREAKING CHANGE` followed by a colon. In the case of hotfix, put `Hotfix` in this part. For other cases, this part should be empty.
+3. [**Recommendation**] If the change can be mapped into a command, then the next part could be the command name, followed by a colon.
+4. [**Recommendation**] Use the right verb with present-tense in original form to descibe what is done:
+  * **Add** for new features.
+  * **Change** for changes in existing functionality.
+  * **Deprecate** for once-stable features removed in upcoming releases.
+  * **Remove** for deprecated features removed in this release.
+  * **Fix** for any bug fixes.
+5. [**Mandatory**] Use period in the end.
 
-This will be the release notes for the next release.
+An example of customer-facing change PR:
 
-e.g.:  
-```
-.. :changelog:
+```[Storage] BREAKING CHANGE: az storage remove: remove --auth-mode argument.```
 
-Release History
-===============
+An example of non-customer-facing change PR:
 
-0.0.3
-+++++
-* This is my customer-facing change.
+```{Aladdin} Add help example for dns.```
 
-0.0.2
-+++++
-* Minor fixes.
+### Format PR Description
 
-0.0.1
-+++++
-* Initial release
-```
+If you would like to write multiple history notes for one PR, please write the notes under `History Notes` section in the PR description,  following the same format described above. In this case, the PR title should be a summary of all the changes in this PR and will not be put into `HISTORY.rst`.
+
+Other than that, you can put any reasonable information in the description above the `History Notes` part.
+
+### Only Modify HISTORY.rst for Hotfix
+As described above, history notes will be auto-generated and insert into `HISTORY.rst` during release. When submitting a normal PR, you don't need to modify `HISTORY.rst` manually. In the case of hotfix, you should create a hotfix branch based on release branch and submit a PR to merge hotfix into release. In this PR, you need to modify `HISTORY.rst` to add history notes for hotfix changes. Later on you also need to merge the hotfix branch into dev branch and the second part of the PR title should be `Hotfix`. The auto-generation of history notes for the next release will ignore the commit that contains `Hotfix`.
```

Comment:
Let's put an example here for easy understanding.

Reply:
added an example.